### DISCUSSION TITLE
Adding folders for sorting plugins

### DISCUSSION THREAD

**Discussion by StumblesCodes** [runelite/runelite#17302] [Ideas]
Default runelite comes with a lot of plugins and turning them off and on is no problem. But, I think it would be beneficial if there was some way to group plugins ( on the runelite app ) by folders.

EXAMPLE: SKILLING folder has 
			wintertodt, tempoross, gotr.
et cetera for pvm, or whatever.

TLDR; Being able to move plugins in to folders for easy sorting in runelite

**Comment by StronkPoE** [comment]
I want to add my voice to this. If it is possible in any way, this should be a feature of runelite by now as we have so many plugins floating about and chucking all the CoX plugins in one place, ToB in another and skilling in another would just make things... neat and easy to deal with?

pleasepleaseplease?

**Comment by ZohiDev** [comment]
Yes please - almost posted my own suggestion but found this one. Navigating the plugin list is a massive PITA and being able to organize it would be amazing.
